mixed_drinks/src/components/AddDrinksButton.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+
+
+class AddDrinksButton extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+
+        }
+    }
+
```

Comment:
If AddDrinksButton is meant to be used with the Input component, then we will have to get rid of this class based component.  You can add a button to the Input component or you can create a generic Button component that is stateless. 
We don't want AddDrinksButton to have state, instead we want to just pass methods as props from a parent component (App.js) and update the parent components state.